README.md

```diff
@@ -1,11 +1,12 @@
 #Nussbaum-Backend
 
 [![Build Status](https://travis-ci.org/ikarulus/Nussbaum-Backend.svg?branch=master)](https://travis-ci.org/ikarulus/Nussbaum-Backend)
+![issue_stats](https://img.shields.io/badge/issues%20closed%20in-about%206%20hours-green.svg?style=flat-square)
 
 Usage: <https://gist.github.com/ikarulus/c57ae21442201fae89fa194c1e021f6d> (de)
 
 ##Install (using Node Package Manager):
 1. `git clone https://github.com/ikarulus/Nussbaum-Backend.git`
 2. `cd` in your "Nussbaum-Backend" directory
-3. `npm install`
+3. `npm install` (if not available simply run `sudo apt-get install nodejs npm`)
```

Comment:
I think everybody who works with this repo is used to npm. Maybe I will reintegrate this later.